### PR TITLE
Add bridge token list

### PR DESCRIPTION
**Pull request by Lbqds**

### CODE REVIEW

File: token-list/utils.ts [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+      tokenId,
+      deployments.groupIndex
+    )
+  }
+
+  const getTokenMetadata = async (tokenId: string): Promise<TokenMetaData> => {
+    if (tokenId === ALPH_TOKEN_ID) {
+      return { name: ALPH.name, symbol: ALPH.symbol, decimals: ALPH.decimals }
+    }
+    const nodeProvider = web3.getCurrentNodeProvider()
```
Comment (h0ngcha0) [inline token-list/utils.ts:43]: Do you think it makes sense to consult [Token List](https://github.com/alephium/token-list) first here?

**Review by h0ngcha0 — COMMENTED** [review]
LGTM :+1: 

Is the next step to publish it somewhere like [this](https://gnosis.blockscout.com/bridged-tokens/eth)?

**Comment by Lbqds** [comment]
> Is the next step to publish it somewhere like [this](https://gnosis.blockscout.com/bridged-tokens/eth)?

Yeah, wormhole uses a dedicated repo to track all bridge tokens: https://github.com/wormhole-foundation/wormhole-token-list

File: token-list/utils.ts (hunk 1 above)
Comment (Lbqds) [reply token-list/utils.ts:49]: Good idea, I have added a check to verify if the token is in the token list, but the token metadata is still queried from the chain.

**Review by h0ngcha0 — APPROVED** [review]
LGTM :+1:

**Review by polarker — APPROVED** [review]
LGTM

**Pull request merged**